### ISSUE TITLE
Get Error “Code 1: Cuda Driver (invalid argument)” when generate tensorrt engine from saved torch model

### ISSUE THREAD

**Issue by sxthunder** [NVIDIA/TensorRT#1427]
## Environment
I build the environment through docker, with the following command:

**TensorRT Version**: 8.0.1.6
**NVIDIA GPU**: GeForce RTX 2080 Ti
**NVIDIA Driver Version**: 430.26
**CUDA Version**: 10.2
**CUDNN Version**: 8
**Operating System**: ubuntu-18.04
**Python Version (if applicable)**: 3.6.9
**Tensorflow Version (if applicable)**: 2.4.1
**PyTorch Version (if applicable)**: 1.8.1

## Relevant Files

I cp a torch bert-base model into the docker.

## Steps To Reproduce

+ clone the repository and build docker image with the following cmd :
```
./docker/build.sh --file docker/ubuntu-18.04.Dockerfile --tag tensorrt-ubuntu18.04-cuda10.2 --cuda 10.2
```
+ build tensorrt oss in the container with following cmd:
```
 cd $TRT_OSSPATH
 mkdir -p build && cd build
 cmake .. -DTRT_LIB_DIR=$TRT_LIBPATH -DTRT_OUT_DIR=`pwd`/out -DCUDA_VERSION=10.2
 make -j$(nproc)
```
+ cp the saved torch bert model, config, vocab into docker
+ cd demo/BERT, run:
```
python builder.py -pt /workspace/bert_dir/bert_base.pth -o ./enignes/bert_large_60.engine -b 1 =s 80 --fp16 -c /workspace/bert_dir/ -v /workspace/bert_dir/vocab.txt
```
The demo code is used for question answer, I changed the code about model parameter name mapping. The model is correctly loaded, but stuck at "engine = builder.build_engine(network, builder_config)".

It is note worthy that when I first get into the container, the "/usr/lib/x86_64-linux-gnu/libcuda.so.1" is linked to "libcuda.so.440.118.02", I can't understand why there is a 440-version of cuda drive. The cuda drive in my host is 430.26. If I directly run the python command mentioned above, it would output the following when import pycuda.autoinit:
```
pycuda._driver.LogicError: cuInit failed: system has unsupported display driver / cuda driver combination
```
To solve this, I relink the "/usr/lib/x86_64-linux-gnu/libcuda.so.1" to "libcuda.so.430.26". After the relink, the above problem was solved, but get the following output:
```
2021-08-03 08:56:05.989816: W tensorflow/stream_executor/platform/default/dso_loader.cc:60] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /usr/local/nvidia/lib:/usr/local/nvidia/lib64:/workspace/TensorRT/build/out:/usr/lib/x86_64-linux-gnu
2021-08-03 08:56:05.989851: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
WARNING:tensorflow:From /usr/local/lib/python3.6/dist-packages/tensorflow/python/compat/v2_compat.py:96: disable_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated and will be removed in a future version.
Instructions for updating:
non-resource variables are not supported in the long term
[TensorRT] INFO: Using configuration file: /workspace/bert_dir/bert_config.json
[TensorRT] INFO: Found 201 entries in weight map
[TensorRT] INFO: [MemUsageChange] Init CUDA: CPU +420, GPU +0, now: CPU 1149, GPU 3730 (MiB)
[TensorRT] INFO: [MemUsageSnapshot] Builder begin: CPU 1368 MiB, GPU 3970 MiB
[TensorRT] INFO: [MemUsageChange] Init cuBLAS/cuBLASLt: CPU +241, GPU +68, now: CPU 1808, GPU 4268 (MiB)
[TensorRT] WARNING: Detected invalid timing cache, setup a local cache instead
[TensorRT] INTERNAL ERROR: [defaultAllocator.cpp::operator()::35] Error Code 1: Cuda Driver (invalid argument)
```

What does this error mean? Is my drive & cuda drive mismatch?

**Comment by sxthunder** [comment]
@cding-nv

**Comment by sxthunder** [comment]
I fixed this problem by build TensorRT 8.0 in cuda 11 or build TensorRT 7.x in cuda 10.2. It seems to be that TensorRT8.0 have some bugs on supporting cuda 10.2
